encoding.bs

```diff
@@ -1236,42 +1246,38 @@ string += decoder.decode(); // end-of-stream</code></pre>
 </dl>
 
 <p>The
-<dfn constructor for=TextDecoder id=dom-textdecoder><code>TextDecoder(<var>label</var>, <var>options</var>)</code></dfn>
-constructor, when invoked, must run these steps:
+<dfn constructor for=TextDecoder lt="TextDecoder(label, options)" id=dom-textdecoder><code>new TextDecoder(<var>label</var>, <var>options</var>)</code></dfn>
+constructor steps are:
 
 <ol>
  <li><p>Let <var>encoding</var> be the result of <a>getting an encoding</a> from <var>label</var>.
 
  <li><p>If <var>encoding</var> is failure or <a>replacement</a>, then <a>throw</a> a {{RangeError}}.
 
- <li><p>Let <var>dec</var> be a new {{TextDecoder}} object.
-
- <li><p>Set <var>dec</var>'s <a for=TextDecoderCommon>encoding</a> to <var>encoding</var>.
+ <li><p>Set <a>this</a>'s <a for=TextDecoderCommon>encoding</a> to <var>encoding</var>.
 
- <li><p>If <var>options</var>'s <code>fatal</code> member is true, then set <var>dec</var>'s
+ <li><p>If <var>options</var>["<code>fatal</code>"] is true, then set <a>this</a>'s
```

Comment:
You can cross-link these with `{{TextDecoderOptions/fatal}}`. Same for all other dictionary members. Maybe you'll want to delay that to a follow-up, though.